### DISCUSSION TITLE
Speed of Migrations

### DISCUSSION THREAD

**Discussion by simonphughes** [hasura/graphql-engine#8561] [Question] [answered]
Just started to hit some issues with running migrations as we move our configuration from dev through to test and production. Our hasura in running in ECS clusters using the Fargate model in AWS, which sits behind a load balancer which handles the usual SSL offloads, etc. 
Running our migration of metadata we are now hitting issues where the load balancer is timing out and returning a http 504 error. Is there anything we can do to speed up the migration of metadata, the command thats causing issues is
hasura metadata apply --endpoint https://myendpoint.com --admin-secret mypassword

Thanks

**Comment by BenoitRanque** [comment, marked as answer]
Possible workaround: try CLI v2.4.0
Reason: we are currently investigating a performance regression in CLI v2.5.0 and up, and have not yet released a fix.
